src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -120,6 +129,21 @@ private void Initialize(Runspace initialRunspace)
             // TODO: Should this be configurable?
             this.SetExecutionPolicy(ExecutionPolicy.RemoteSigned);
 
+            var psVersionTable = this.currentRunspace.SessionStateProxy.GetVariable("PSVersionTable") as Hashtable;
```

Comment:
If so I'd say make it into its own private function, something like GetPowerShellVersion